management/src/main/java/org/kaazing/gateway/management/context/ManagementGatewayListener.java

```diff
@@ -69,8 +92,8 @@ public void startingService(ServiceContext serviceContext) {
     }
 
     @Override
-    public void stoppedGateway() {
```

Comment:
The ManagementGatewayListener initially contained the following code:
@Override
public void stoppedGateway() {
	managementContext.close();
}
In DefaultManagementContext (implementation of ManagementContext), the close method performed a close on the monitoringEntityFactory.

However, due to the fact that in the ManagementGatewayListener the following method was added (performing a cleanup on the monitoringEntityFactory):
@Override
public void stoppedService(ServiceContext serviceContext) {
	serviceContext.getMonitoringFactory().close();
}
I have removed the stoppedGateway method in ManagementGatewayListener and the close method from ManagementContext (and its implementation, DefaultManagementContext).

Additional information on this can be obtained from ManagementGatewayListener, ManagementContext and DefaultManagementContext Git logs. Please give me your feedback on this.